features/miniAlert.js

```diff
@@ -0,0 +1,25 @@
+//Thanks Vals
+function inDungeons() {
+    if(!(Scoreboard.getTitle().trim() != "" && Scoreboard.getLines().length > 0)) {
+        return;
+    }
+    for(let i = 0; i < Scoreboard.getLines().length; i++) {
+        let line = Scoreboard.getLines()[i]
+        if(line.getName().includes('§cThe Cata') && line.getName().includes('mbs §8('))

```

Comment:
use double quotes since the whole project uses those so you should too

Reply:
Didn't Ask.